Documentation/EyeTracking/EyeTracking_BasicSetup.md

```diff
@@ -72,15 +80,17 @@ In general, it is recommended to avoid showing an eye gaze cursor or if absolute
 We do recommend to hide the default head gaze cursor that is attached to the MRTK gaze pointer profile by default.
     - Navigate to your MRTK configuration profile -> _'Input'_ -> _'Pointers'_
     - Clone the _'DefaultMixedRealityInputPointerProfile'_ to make changes to it.
-    - At the top of the _'Pointer Settings'_, you should assign an invisible cursor prefab to the _'GazeCursor'_. If you downloaded the MRTK Examples folder, you can simply reference the included _'EyeGazeCursor'_ prefab.
+    - At the top of the _'Pointer Settings'_, you should assign an invisible cursor prefab to the _'GazeCursor'_. You can do this by selecting the _'EyeGazeCursor'_ prefab from the MRTK Foundation.
 
 ### Enabling eye-based gaze in the gaze provider
 
 In HoloLens v1, head gaze was used as primary pointing technique.
-While head gaze is still available via the _GazeProvider_ in MRTK which is attached to your [Camera](https://docs.unity3d.com/ScriptReference/Camera.html), you can check to use eye gaze instead by ticking the _'IsEyeTrackingEnabled'_ checkbox in the gaze settings of the input pointer profile.
+While head gaze is still available via the MRTK _GazeProvider_ MonoBehaviour which is attached to your [Camera](https://docs.unity3d.com/ScriptReference/Camera.html), you can use eye gaze instead by ticking the _'UseEyeTracking'_ checkbox.
```

Comment:
I believe this flow has changed a bit as of https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7246

Reply:
![image](https://user-images.githubusercontent.com/3580640/78585194-46aebd00-77ee-11ea-8ff4-42608e2fe5e5.png)

Reply:
Well I'll be. I could swear I couldn't find that setting, but now I see it there plain as day. Thanks @keveleigh !

I pushed an update, reverting my doc changes related to  IsEyeTrackingEnabled.